### PR TITLE
added branch support : ECO-2743

### PR DESCRIPTION
**Pull request by MaitriGurey1**
1. added branch support
2. updated env file accordingly

### CODE REVIEW

**Comment by surajcontentstack** [comment]
@MaitriGurey1 PR title does not have ticket ID

File: plugins/contentstack.js [hunk 1]
```diff
@@ -7,6 +7,9 @@ const Stack = contentstack.Stack({
   api_key: process.env.CONTENTSTACK_API_KEY,
   delivery_token: process.env.CONTENTSTACK_DELIVERY_TOKEN,
   environment: process.env.CONTENTSTACK_ENVIRONMENT,
+  branch: process.env.CONTENTSTACK_BRANCH
+    ? process.env.CONTENTSTACK_BRANCH
+    : 'main',
   region: process.env.CONTENTSTACK_REGION
```
Comment (surajcontentstack) [inline plugins/contentstack.js:13]: this was already present?

**Review by surajcontentstack — CHANGES_REQUESTED** [review]

**Review by Amitkanswal — COMMENTED** [review]
Pls, check if we have an update in dependencies and add a version bump to package.json

File: plugins/contentstack.js (hunk 1 above)
Comment (MaitriGurey1) [reply plugins/contentstack.js:13]: Hi @surajcontentstack , region support was already present, I have added the branch support, few of the starters already have region support(I have the list)

**Review by surajcontentstack — APPROVED** [review]

**Review by Amitkanswal — APPROVED** [review]

**Pull request merged**